src/GoogleMapMarker.svelte

```diff
@@ -0,0 +1,15 @@
+<script>
+  import { getContext } from "svelte";
```

Comment:
I haven't enabled linting here yet but the code style across the rest of the project is single quotes and no semi-colons.